provisioner.go

```diff
@@ -342,20 +352,34 @@ func (p *LocalPathProvisioner) createHelperPod(action ActionType, cmdsForPath []
 				{
 					Name:    "local-path-" + string(action),
 					Image:   p.helperImage,
-					Command: append(cmdsForPath, filepath.Join("/data/", volumeDir)),
+					Command: cmdsForPath,
+					Args:    []string{filepath.Join(parentDir, volumeDir), strconv.FormatInt(sizeInBytes, 10)},
```

Comment:
It's better if we can use `--size` parameter instead of piling them up, otherwise it's easy to mix up the sequence.

Reply:
Also, this changes the input of the setup/teardown script format. We need to document the change.